### PR TITLE
Scouting Drones for Nukeops [WIP]

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request

Adds scouting drones to the traitor module (for nukeops only).
#341 - KMC's idea.

Currently they are black colored drones which can run around the station, seeping in and out of airlocks, and can be controlled from distances with the syndicates controller.

To be able to control them you need to sit still with your drone controller. They operate like normal drones for the most part, but cannot pick up anything. You can also store pinpointers in them which can be used to locate the disk, etc. They can see through walls like normal drones.

Take into account that it's not a good idea to let humans cross these, and never to let borgs cross you either. Humans have a random chance of stunning them, and borgs will stun them always if they manage to bump into you.

Do you think these guys should be faster than the average drone? It's currently a speed of 0 which is the normal drone speed, and -1 is pretty insane.

I could use some feedback on this or any ideas that you guys may have for them.

One thing that I would love to cover is to give them action buttons, for when the user is controlling their drone.

[WIP] is for the icons. They aren't... very proportional. It should be fixed soon @Kmc2000

### CODE REVIEW

**Comment by oisin100** [comment]
> Do you think these guys should be faster than the average drone? It's currently a speed of 0 which is the normal drone speed, and -1 is pretty insane.

Cant you use floating points?

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -554,6 +554,14 @@ var/list/uplink_items = list()
 	item = /obj/item/clothing/glasses/thermal/syndi
 	cost = 6
 
+/datum/uplink_item/device_tools/scoutdrone
+	name = "Syndicate Scouting Drone"
+	desc = "A survalliance drone which is known best for scouting enemies bases and trying to discover vital information which can be used for a team's objective."
```
Comment (oisin100) [inline code/datums/uplink_item.dm:559]: Surveillance*

**Comment by oisin100** [comment]
Also, Can we have the ability to teleport them in from the Syndicate base so that the nuke ops can spy on the station and cause minor havoc whilst they are waiting till they are allowed to go to the station? Maybe give them a 1 use teleport option what teleports them to a random carp spawn.

RAINBOW SIX SIEGE CONFIRMED!

**Comment by ShadowDeath6** [comment]
> stepped on by someone and stunned
> stomped to death
> wasted tc

no stunning please, drones already get fucked by telebatons

**Comment by ghost** [comment]
these are legitimately shit

it's declaring war without the TC bonus because if one guy sees it 'OPS!!!!!!!' shuttle is called and cap is fortified in the brig.

stunning them is also shit.

so either make them cost like 2 TC for how useless they are or give them some legitimate utility and make them not an instant scream of OPSSSSSSSSSSS@!!!!!!!!!!!!!!!!!!!!!!!!!!11

**Comment by Super3222** [comment]
I'm open into suggestions for these guys.

@AdamElTablawy, I think the camouflage ability we talked about earlier would be a pretty nice feature for these. Though would definitely raise the price.

@Oisin100, that sounds like fun but there could be some issues. One being the teleporter is still a viable option in transportating them.

**Comment by ShadowDeath6** [comment]
The teleporter is 40 TC and not worth it if you're going to be stealthy since that takes a massive chunk out of your budget.

**Comment by FluffySurvivor** [comment]
No, don't make them nuke op only. It's literally a dead giveaway there are nuke ops and unless it's your first time playing the game there is no point in "scouting". 

There is nothing to scout. 

So yeah, it's crap as it is atm. 

My suggestion is to allow them to be remotely detonated, with a small blast KO'ing only the target on top of it. THAT would be interesting. 

@ShadowDeath6 you can get it for free from the abandoned sat

**Comment by ShadowDeath6** [comment]
> free from the abandoned sat

SHHHHH

**Comment by Kmc2000** [comment]
I said they were meant to be a traitor item

For exactly the reasons adam is raging about

But yeah EWWW STUNS

If you see them you are meant to just stomp on them, not OGODSTUNRAEP

Also, whaddya mean out of proportion? i fixed that

**Comment by Kmc2000** [comment]
Also, i said 3tc 

> :^ )

**Comment by ShadowDeath6** [comment]
Just make them completely invisible :^)

**Comment by Super3222** [comment]
> Also, whaddya mean out of proportion? i fixed that

Yeah I commit this before I got em, this PR currently holds the old icons. It'll be changed in the next commit.

Camouflage ability is in the works.

**Comment by Super3222** [comment]
I'm going to go ahead and close this, since I have already rebased the commits.

**Pull request closed without merging**